File: src/Learn/Arrays101/removeDuplicatesFromSortedArray.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int length = 0;
        int currNum;
        for(auto num : nums){
        	if(num == currNum) continue;
        	else{
        		currNum = num;
        		nums[length++] = currNum;
        	}
        }
        return length;
    }
};

File: src/Learn/Arrays101/RemoveDuplicatesFromSortedArray.cs
public class Solution {
    public int RemoveDuplicates(int[] nums) {
    	int n = nums.Length;
    	if(n == 0) return 0;
        int j = 1;
        for(int i = 1; i < n; i++){
        	if(nums[i] != nums[i - 1]) nums[j++] = nums[i];
        }
        return j;
    }
}